vars/runFunctionalTest.groovy

```diff
@@ -0,0 +1,112 @@
+    fi
+    if [[ -n "${params.k8s_scylla_operator_chart_version ? params.k8s_scylla_operator_chart_version : ''}" ]] ; then
+        export SCT_K8S_SCYLLA_OPERATOR_CHART_VERSION=${params.k8s_scylla_operator_chart_version}
+    fi
+
+    if [[ -n "${params.scylla_mgmt_agent_version ? params.scylla_mgmt_agent_version : ''}" ]] ; then
+        export SCT_SCYLLA_MGMT_AGENT_VERSION=${params.scylla_mgmt_agent_version}
+    fi
+
+    if [[ -n "${params.scylla_mgmt_agent_repo ? params.scylla_mgmt_agent_repo : ''}" ]] ; then
+        export SCT_SCYLLA_MGMT_AGENT_REPO=${params.scylla_mgmt_agent_repo}
+    fi
+
+    if [[ -n "${params.scylla_ami_id ? params.scylla_ami_id : ''}" ]] ; then
+        export SCT_AMI_ID_DB_SCYLLA="${params.scylla_ami_id}"
+    elif [[ -n "${params.gce_image_db ? params.gce_image_db : ''}" ]] ; then
+        export SCT_GCE_IMAGE_DB="${params.gce_image_db}"
+    elif [[ -n "${params.scylla_version ? params.scylla_version : ''}" ]] ; then
+        export SCT_SCYLLA_VERSION="${params.scylla_version}"
+    elif [[ -n "${params.scylla_repo ? params.scylla_repo : ''}" ]] ; then
+        export SCT_SCYLLA_REPO="${params.scylla_repo}"
+    elif [[ "${params.backend ? params.backend : ''}" == *"k8s"* ]] ; then
+        echo "Kubernetes backend can have empty scylla version. It will be taken from defaults of the scylla helm chart"
+    else
+        echo "need to choose one of SCT_AMI_ID_DB_SCYLLA | SCT_SCYLLA_VERSION | SCT_SCYLLA_REPO | SCT_GCE_IMAGE_DB"
+        exit 1
+    fi
+
+    if [[ -n "${params.oracle_scylla_version ? params.oracle_scylla_version : ''}" ]] ; then
+        export SCT_ORACLE_SCYLLA_VERSION="${params.oracle_scylla_version}"
+    fi
+
+    if [[ -n "${params.gemini_seed ? params.genini_seed : ''}" ]] ; then
+        export SCT_GEMINI_SEED="${params.gemini_seed}"
+    fi
+
+    export SCT_POST_BEHAVIOR_DB_NODES="${params.post_behavior_db_nodes}"
+    export SCT_POST_BEHAVIOR_LOADER_NODES="${params.post_behavior_loader_nodes}"
+    export SCT_POST_BEHAVIOR_MONITOR_NODES="${params.post_behavior_monitor_nodes}"
+
+    if [[ -n "${params.provision_type ? params.provision_type : ''}" ]] ; then
+        export SCT_INSTANCE_PROVISION="${params.provision_type}"
+    fi
+
+    if [[ -n "${params.instance_provision_fallback_on_demand ? params.instance_provision_fallback_on_demand : ''}" ]] ; then
+        export SCT_INSTANCE_PROVISION_FALLBACK_ON_DEMAND="${params.instance_provision_fallback_on_demand}"
+    fi
+    export SCT_AMI_ID_DB_SCYLLA_DESC=\$(echo \$GIT_BRANCH | sed -E 's+(origin/|origin/branch-)++')
+    export SCT_AMI_ID_DB_SCYLLA_DESC=\$(echo \$SCT_AMI_ID_DB_SCYLLA_DESC | tr ._ - | cut -c1-8 )
+    if [[ "${params.update_db_packages || false}" == "true" ]] ; then
+        export SCT_UPDATE_DB_PACKAGES="${params.update_db_packages}"
+    fi
+
+    if [[ -n "${params.tag_ami_with_result ? params.tag_ami_with_result : ''}" ]] ; then
+        export SCT_TAG_AMI_WITH_RESULT="${params.tag_ami_with_result}"
+    fi
```

Comment:
The same here. Do we really need it?

Reply:
This is not specific for k8s, it is general functional suite

Reply:
Same as above, must not have been "general" suite. "dtest" exists for such case.